No asITypeInfo forward declaration in ScriptInstance.h

When directly including `Urho3D/AngelScript/ScriptInstance.h` without any of the other Script headers before it, I get this error:
```
/home/nathan/Projects/Urho3D/Build/include/Urho3D/AngelScript/ScriptInstance.h:41:36: error: variable or field ‘CleanupTypeInfoScriptInstance’ declared void
   41 | void CleanupTypeInfoScriptInstance(asITypeInfo *type);
      |                                    ^~~~~~~~~~~
/home/nathan/Projects/Urho3D/Build/include/Urho3D/AngelScript/ScriptInstance.h:41:36: error: ‘asITypeInfo’ was not declared in this scope; did you mean ‘TypeInfo’?
   41 | void CleanupTypeInfoScriptInstance(asITypeInfo *type);
      |                                    ^~~~~~~~~~~
      |                                    TypeInfo
/home/nathan/Projects/Urho3D/Build/include/Urho3D/AngelScript/ScriptInstance.h:41:49: error: ‘type’ was not declared in this scope
   41 | void CleanupTypeInfoScriptInstance(asITypeInfo *type);
      |                                                 ^~~~
```

--- a/Source/Urho3D/AngelScript/ScriptInstance.h
+++ b/Source/Urho3D/AngelScript/ScriptInstance.h
@@ -30,6 +30,7 @@
 class asIScriptFunction;
 class asIScriptObject;
 class asIScriptContext;
+class asITypeInfo;
 
 namespace Urho3D
 {

